app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -20,11 +20,16 @@ def self.description
     @description ||= "Microsoft System Center VMM".freeze
   end
 
-  def self.raw_connect(connect_params)
+  def self.raw_connect(connect_params, validate = false)
     require 'winrm'
 
     connect_params[:operation_timeout] = 1800
```

Comment:
This would ignore the existing parameter, wouldn't it? Should it be `||= 1800`? Or do you deliberately want it set to 1800? Same for password.

Reply:
@djberg96 good catch. Will still want to override password though to ensure it's decrypted